app/frontend/components/pages/delegations/delegatePage.js

```diff
@@ -0,0 +1,58 @@
+const {h} = require('preact')
+const connect = require('unistore/preact').connect
+const actions = require('../../../actions')
+
+const DelegatePage = ({
+  poolAddress,
+  updatePoolAddress,
+  setAdalitePool,
+  submitDelegation,
+  currentStakepool,
+}) => {
+  return h(
+    'div',
+    {class: 'delegate card'},
+    h('h2', {class: 'card-title'}, 'Delegate Stake'),
+    h('input', {
+      type: 'text',
+      id: 'pool-address',
+      class: 'input pool-address fullwidth',
+      name: 'pool-address',
+      placeholder: 'Stakepool Address',
+      value: poolAddress,
+      onInput: updatePoolAddress,
+      autocomplete: 'off',
+    }),
+    h(
+      'button',
+      {
+        class: 'button delegate',
+        id: 'add-pool',
+        onClick: setAdalitePool,
+        disabled: false,
+      },
+      'AdaLite stake pool'
+    ),
+    h(
+      'div',
+      {class: 'validation-row'},
+      h(
+        'button',
+        {
+          class: 'button primary',
+          disabled: currentStakepool.address !== '',
```

Comment:
![image](https://user-images.githubusercontent.com/22454337/69374993-952c4680-0ca7-11ea-898c-40a192e438bc.png)
I propose showing a tooltip saying 'Already delegated' or something like that over the disabled button, its not important, just a nice thing to have. Find inspiration here:
![image](https://user-images.githubusercontent.com/22454337/69375075-bc831380-0ca7-11ea-9844-e1a10420fe62.png)

Reply:
Tooltip added.